intro-HPC/ch_introduction.tex

```diff
@@ -285,11 +340,19 @@ \subsection{What programming languages can I use?}
 M(VA)PICH, OpenMPI.
 \fi
 \ifleuven
-Commonly used software packages are ABINIT, CP2K, Gaussian, Gromacs, Molpro,
-NWChem, Quantum Espresso, R, Siesta, TURBOMOLE, WIEN2k, \ldots
-
-Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI,
-M(VA)PICH, OpenMPI.
+Commonly used software packages are:
+\begin{itemize}
+\item{in bioinformatics: beagle, Beast, bedtools, bowtie, BWA, Mr. Bayes, TopHat, TRIQS,}
+\item{in chemistry: Gaussian, GROMACS, Molpro, NAMD, NWChem, Siesta, Turbomole, VMD,}
+\item{in engineering: Abaqus, Ansys, Comsol,}
```

Comment:
I understood that it was meant to be site-specific (see visualization section @KUL). Otherwise users may expect that software is already installed. But I will wait for the others to comment on it as well...